### PR TITLE
chore: update PassportConnect

### PR DESCRIPTION
**Pull request by thelostone-mc**
##### Description

This PR is being merged to passport-integration-branch

- update design of passport connect page
- update calculate button 

refs #977 
refs #974

### CODE REVIEW

**Review by melissa-neira — CHANGES_REQUESTED** [review]
Hey, this doesn't include the latest copy changes made. Please update :)

**Comment by melissa-neira** [comment]
Also, something weird is happening to my Passport score. I'm not sure if this is related to this PR or not.
https://www.loom.com/share/70656ca576b242388a6ca5f5d6381ef3

**Comment by thelostone-mc** [comment]
@melissa-neira could you hit recalculate score

File: packages/grant-explorer/src/features/common/PassportBanner.tsx [hunk 1]
```diff
@@ -43,6 +43,12 @@ export default function PassportBanner(props: {chainId?: string, roundId?: strin
         const res = await fetchPassport(address, PASSPORT_COMMUNITY_ID);
         if (res.ok) {
           const json = await res.json();
+
+          if (json.status == "PROCESSING") {
+            await callFetchPassport();
```
Comment (vacekj) [inline packages/grant-explorer/src/features/common/PassportBanner.tsx:48]: won't this result in a possibly infinite loop? If it never changes from processing. Also, we should probably have exponential backoff here. I'll merge anyway, but I'll also add a TODO here to address this.

**Pull request merged**